segments/helpers.py

```diff
@@ -0,0 +1,144 @@
+import logging
+import redis
+
+from django.contrib.auth import get_user_model
+from django.db import connections
+from segments import app_settings
+
+logger = logging.getLogger(__name__)
+
+
+class SegmentHelper(object):
+    segment_key = 's:%s'
+    segment_member_key = 'sm:%s'
+    segment_member_refresh_key = 'sm:refresh'
+
+    def __init__(self):
+        self.redis = None
+        self.user_key = 'sm:%s'
+        self.refresh_key = 'sm:refresh'
+
+    def get_redis(self):
+        if not self.redis:
+            self.redis = redis.StrictRedis.from_url(app_settings.SEGMENTS_REDIS_URI)
+        return self.redis
```

Comment:
Property seems better for that case, no need to call the method everywhere

```suggestion
        @property
        def redis(self):
            if not self.__redis
            self.__redis = redis.StrictRedis.from_url(app_settings.SEGMENTS_REDIS_URI)
        return self.__redis
```

Reply:
Or just instantiate the attribute in `__init__.py`

Reply:
I'm planning to inject fakeredis into this for the tests.  This seemed a clearer way to go about it, and without instantiating a connection before it's actually used.